flight/src/tests.rs

```diff
@@ -22,25 +20,24 @@ fn test_conclusion_events_to_record_batch() {
         ConclusionEvent::Data(ConclusionData {
             id: Cid::default(),
             init: ConclusionInit {
-                stream_type: StreamType::Tile,
+                stream_type: 0,
```

Comment:
I actually think using named StreamTypes in test code is still nice and improves readability (assuming we have an existing type we can re-use, probably not worth adding one just for use in the tests). I'm only concerned about keeping the concrete streamtypes out of the production code, not the test code.

We should probably use MIDs in our tests though not TileDocuments since C1 doesn't actually support TileDocuments.